### PR TITLE
Enhancement: Update chart cursor component to allow more flexible labels

### PR DESCRIPTION
**Pull request by znicholasbrown**
No change to defaults here, just allowing more flexible styling

### CODE REVIEW

File: src/components/ChartCursor/ChartCursor.vue [hunk 1]
```diff
@@ -2,13 +2,17 @@
   <div ref="chart" class="chart-cursor" v-bind="{ onPointerenter, onPointermove, onPointerleave }">
     <slot />
     <template v-if="showCursor">
-      <div class="chart-cursor__cursor" :style="cursorStyles" />
+      <slot name="cursor" v-bind="{ cursor }">
+        <div class="chart-cursor__cursor" :style="cursorStyles" />
+      </slot>
```
Comment (pleek91) [inline src/components/ChartCursor/ChartCursor.vue:7]: Whats the thought behind overriding the cursor itself? I imagined we would always want this to look the same so users have a consistent visual queue.

File: src/components/ChartCursor/ChartCursor.vue [hunk 2]
```diff
@@ -165,8 +175,6 @@
   shadow-lg
   p-2
   rounded-default
-  z-10;
-  top: calc(100% + theme('spacing.2'));
```
Comment (pleek91) [inline src/components/ChartCursor/ChartCursor.vue:169]: additionally you probably don't need `absolute` on this anymore right?

**Review by pleek91 — APPROVED** [review]

File: src/components/ChartCursor/ChartCursor.vue (hunk 1 above)
Comment (znicholasbrown) [reply src/components/ChartCursor/ChartCursor.vue]: I have a use case in mind but I realized I could probably override directly via CSS instead, which I think would be a little more straightforward. Will revisit if I find it's not, for now I reverted this change 👍🏻

File: src/components/ChartCursor/ChartCursor.vue (hunk 2 above)
Comment (znicholasbrown) [reply src/components/ChartCursor/ChartCursor.vue:169]: Good call, fixed

File: src/components/ChartCursor/ChartCursor.vue (hunk 1 above)
Comment (pleek91) [reply src/components/ChartCursor/ChartCursor.vue]: Yeah if we want to do something like customize the color or something a custom property might feel nice here.

Comment (znicholasbrown) [reply src/components/ChartCursor/ChartCursor.vue]: mm yeah good call

**Pull request merged**